src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -156,36 +161,21 @@ def create_llm_inputs(
             input_type, dataset_name, starting_index, length, tokenizer
         )
 
-        if input_type == PromptSource.DATASET:
```

Comment:
Pulled these out to be their own function get_generic_dataset_json. 

his could have been its own tiny PR and will be in the future. In this case, since this was already a clone of another branch for a closed PR, I wanted to avoid creating conflicting commits in the unlikely case we use the old PR/branch and that causes a conflict.